source/renderer/app/stores/VotingStore.ts

```diff
@@ -423,23 +423,21 @@ export default class VotingStore extends Store {
     const { network, isMainnet } = this.environment;
     const intl = i18nContext(currentLocale);
 
-    try {
-      await votingPDFGenerator({
-        nextVotingFundNumber,
-        qrCode,
-        walletName,
-        currentLocale,
-        currentDateFormat,
-        currentTimeFormat,
-        desktopDirectoryPath,
-        network,
-        isMainnet,
-        intl,
-      });
-      // @ts-ignore ts-migrate(2554) FIXME: Expected 1 arguments, but got 0.
+    const wasSaved = await votingPDFGenerator({
+      nextVotingFundNumber,
+      qrCode,
+      walletName,
+      currentLocale,
+      currentDateFormat,
+      currentTimeFormat,
+      desktopDirectoryPath,
+      network,
+      isMainnet,
+      intl,
+    });
+
+    if (wasSaved) {
       this.actions.voting.saveAsPDFSuccess.trigger();
-    } catch (error) {
-      throw new Error(error);
```

Comment:
Is removing the error handling safe here? How would the user know that an error occurred?

Reply:
This try-catch block doesn't do anything. If you're catching the error, not doing anything with it, and immediately rethrowing it, then the effect is exactly the same as not having the try-catch block at all :)

Reply:
It would make sense to keep this try-catch block if we were throwing a more concrete error instead or providing a different error message, but as it stands it's just adding noise :)

Reply:
That's true indeed 👍

Reply:
although you could also say the extra return values added noise 😋 you could just have thrown in the `votingPDFGenerator` instead of returning false … btw. I think `true` and `false` are not necessarily great return values for "did it work?"

Reply:
or let's re-phrase my last comment: Boolean would be a good return value if the function just returned the information about success (eg. `wasPDFSaved()`). But since this function has side effects (is doing something, not just returning the status), I think throwing is also reasonable

Reply:
The booleans are actually not indicating whether it worked or failed. It indicates whether the user actually saved the file or cancelled the process using the "Cancel" button. But I can see it is confusing, see my latest commit - would enum be more readable?

Reply:
Ah got it, so then throwing is less reasonable 👍  … yes I guess enums could improve the readability of return value